### DISCUSSION TITLE
Adding semantic releases to Directus

### DISCUSSION THREAD

**Discussion by juancarlosjr97** [directus/directus#4419] [Feature Requests]
Hello, I have been working recently on how to structure even more my projects and during the last few months, I added semantic-releases to my projects.

What do you think if we added semantic -releases to Directus?

**Benefits**

- Automatic changelog
- Tag release automatically
- Clearer commit structure

I would be happy to add it if you like the idea :)

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/discussions/3572
